lib/standalone.mjs

```diff
@@ -60,11 +69,16 @@ export default async function standalone({
 	reportInvalidScopeDisables,
 	reportNeedlessDisables,
 	reportUnscopedDisables,
+	suppressAll,
+	suppressRule,
+	suppressLocation,
+	suppressPrune,
 	validate = true,
 }) {
 	const startTime = Date.now();
 
-	const hasOneValidInput = (files && !isString(code)) || (!files && isString(code));
+	const useStdin = !files && isString(code);
```

Comment:
[suggestion] I don't think the name `useStdin` is accurate in the context of `standalone()` since it can be used as a Node.js API, not CLI.

So, how about renaming it like so?

```suggestion
	const useInputCode = !files && isString(code);
```